test/scripts/jenkins_xpack_visual_regression.sh

```diff
@@ -21,3 +21,6 @@ yarn percy exec -t 10000 -- -- \
 
 # cd "$KIBANA_DIR"
 # source "test/scripts/jenkins_xpack_page_load_metrics.sh"
+
+cd "$XPACK_DIR"
+source "$KIBANA_DIR/test/scripts/jenkins_xpack_saved_objects_field_metrics.sh"
```

Comment:
isn't `test/scripts/jenkins_xpack_visual_regression.sh` going to be called outside of the baseline job?

If so, `jenkins_xpack_saved_objects_field_metrics.sh` is run elsewhere, so would run twice

Reply:
It isn't currently, and we won't be re-enabling the visual regression tests on master/PRs until after I've refactored the baseline job (and a few other blockers are resolved) so I don't think it's a serious issue.

Reply:
Definitely crufty though